libraries/script-engine/src/AbstractScriptingServicesInterface.h

```diff
@@ -8,7 +8,8 @@
 //  Distributed under the Apache License, Version 2.0.
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
-
+/** @addtogroup ScriptEngine */
+/*@{*/
```

Comment:
```suggestion
/** @addtogroup ScriptEngine
 *  @{
 */
```
This, along with the change at the end of the file, makes the class appear in the Modules section.